torch/quantization/fake_quantize.py

```diff
@@ -280,6 +280,8 @@ def __init__(
         self.register_buffer("observer_enabled", torch.tensor([0], dtype=torch.long))
         self.is_symmetric_quant = _is_symmetric_quant(self.activation_post_process.qscheme)
 
+        self.quant_min, self.quant_max = self.activation_post_process.quant_min, self.activation_post_process.quant_max
```

Comment:
is this guaranteed to be executed after `self.activation_post_process` is initialized, in all cases?

Reply:
qq, does it make sense to take `quant_min/quant_max` as constructor arguments, if they are overriden by this line of code?

Reply:
> is this guaranteed to be executed after `self.activation_post_process` is initialized, in all cases?

Yes, it should be since we call the constructer of the observer before this and I moved the quant_min/max calculation to the observer init function

> does it make sense to take quant_min/quant_max as constructor arguments, if they are overriden by this line of code?

It doesn't. I noticed this is how the current FakeQuantize modules do it too, even though we have a quant_min/max argument passed in the ultimate thing that gets used is the one set by the observer. It feels quite confusing to me.